src/main/resources/assets/structurize/gui/windowswitchpack.xml

```diff
@@ -3,9 +3,13 @@
 
     <button id="cancel" source="structurize:textures/gui/buildtool/button_medium.png" label="$(structurize.gui.buildtool.cancel)" size="86 17" pos="3 3" textcolor="black" texthovercolor="gray"/>
     <input id="filter" size="129 17" pos="150 3" maxlength="25" visible="true"/>
+    <text id="packname" pos="0 0" size="119 20" color="white" textscale="1.5"/>
 
-    <list size="420 200" pos="0 20" id="packs" visible="false">
-        <box size="100% 100" linewidth="0">
+    <list size="420 200" pos="0 40" id="packs" visible="false">
+        <box size="100% 100" pos="0 20" linewidth="0">
```

Comment:
Box with zero linewidth is view...

Reply:
We always use boxes for the inner list boxing.

Reply:
True, but shouldn't matter imho